src/model/core/discharge/discharge.jl

```diff
@@ -27,9 +27,8 @@ function discharge!(EP::Model, inputs::Dict, setup::Dict)
 
 	## Objective Function Expressions ##
 
-	# Variable costs of "generation" for resource "y" during hour "t" = variable O&M plus fuel cost
-	@expression(EP, eCVar_out[y=1:G,t=1:T], (inputs["omega"][t]*(dfGen[y,:Var_OM_Cost_per_MWh]+inputs["C_Fuel_per_MWh"][y,t])*vP[y,t]))
-	#@expression(EP, eCVar_out[y=1:G,t=1:T], (round(inputs["omega"][t]*(dfGen[y,:Var_OM_Cost_per_MWh]+inputs["C_Fuel_per_MWh"][y,t]), digits=RD)*vP[y,t]))
+	# Variable costs of "generation" for resource "y" during hour "t" = variable O&M, the fuel costs will be determined in fuel.jl
```

Comment:
remove the part after the comma as this is only describing a change to the code, not the current functionality. or, include a note that fuel costs are added separately. either way the current wording is talking about the change, not the functionality

Reply:
ok I removed that